model-optimizer/mo/utils/broadcasting.py

```diff
@@ -124,7 +124,6 @@ def bi_directional_broadcasting(input_value: np.array, second_shape: np.array):
                                                                                                second_shape)
     return np.array(input_value * np.ones(second_shape), dtype=input_value.dtype)
 
-
 def explicit_broadcasting(input_value: np.array, target_shape: np.array, axes_mapping: np.array) -> np.array:
```

Comment:
There should be exactly 2 lines between functions

Reply:
fixed